### PR TITLE
Adds monkey cubes to the biogen for 250 points each

### PR DESCRIPTION
**Pull request by Intigracy**
self explanatory

### CODE REVIEW

File: code/modules/hydroponics/biogenerator.dm [hunk 1]
```diff
@@ -41,6 +41,13 @@
 	other_amounts=list(5)
 	result=/obj/item/weapon/reagent_containers/food/snacks/meat
 
+/datum/biogen_recipe/food/monkeycube
+	id="monkeycube"
+	cost=500
+	name="monkey cube"
+	other_amounts=list(5)
```
Comment (ComicIronic) [inline code/modules/hydroponics/biogenerator.dm]: What does this mean?

Comment (Iamgoofball) [inline code/modules/hydroponics/biogenerator.dm]: you can make 5x monkey cubes for 500 X 5

Comment (Intigracy) [inline code/modules/hydroponics/biogenerator.dm]: What he said.

**Comment by Probe1** [comment]
thats pretty big.  i don't think i'll waste my time doing anything except this if cargo has at least one person or we have an AI.

**Comment by Intigracy** [comment]
Now it's 250.

**Comment by Probe1** [comment]
I'm happy.  It's now comparable to the best biogen plant (eggplants) but doesn't completely negate them.  A properly spliced eggplant will net you five eggplants (20 biogen points; 100 total) every 30 seconds.  You can plant as many as you like without spending any credits and can get more biogen fuel than you'll ever need in short order.

With this addition, people other than botanists stand a fair chance at being on the level with good botanists.  Unlike good botanists, it will cost 20 credits and the player will have to go through cargo so there's a 2 minute delay + time to order and process order.  While 250 may equal 12 eggplants on their own and that's a hefty amount, it'll be 5 monkey cubes per trip for 1250 biogen points and two monkey cube crates will be enough to last a shift for the chef or bartender.

1250 points (1 crate of cubes @250 biogen points) gets you:
500 units of milk
25 slabs of meat (no one uses this when synthmeat and dead diona nymphs are free)
More nutrients bottles than you'd ever need
12 wallets
5 pairs of leather gloves 
4 toolbelts
3-4 bags of various types
and 5-25 of other items that no one ever uses the biogen for.

So just one monkey cube crate at 250 per will be extremely lucrative but require you to go through cargo which is never a sure thing.  It's on par with eggplants and has more but is slower to acquire.  For chefs that only want milk or some maybe a scientist that just wants gadget bags, this will be a very nice addition.

**Comment by Probe1** [comment]
Post merge note:  Every post I've made assumed "Adds" meant putting monkey cubes INTO the biogenerator.  

It in fact works the opposite way.  This PR "Includes" monkey cubes as a PRODUCT of the biogenerator.

I am a dumb.  Fortunately I think the 250 value is still a good number.

**Pull request merged**